src/crl/types.rs

```diff
@@ -30,6 +30,19 @@ pub enum CertRevocationList<'a> {
     Borrowed(BorrowedCertRevocationList<'a>),
 }
 
+#[cfg(feature = "alloc")]
+impl From<OwnedCertRevocationList> for CertRevocationList<'_> {
```

Comment:
I wrote a comment for the previous commit lamenting the lack of these. I think these are important and should be in the previous commit, because IMO ~all of the `CertRevocationList` construction sites should use them instead of manually specifying a variant to construct.

Reply:
Fixed up throughout, with the exception of a couple places where it wasn't easy to use (e.g. in `client_auth_revocation.rs` when making a `&[&CertRevocationList]`.